src/Compilers/Core/Portable/TreeDumper.cs

```diff
@@ -106,16 +108,16 @@ private void DoDumpCompact(TreeDumperNode node, string indent)
             }
         }
 
-        public static string DumpXML(TreeDumperNode root, string indent = null)
+        public static string DumpXML(TreeDumperNode root, string? indent = null)
         {
             var dumper = new TreeDumper();
-            dumper.DoDumpXML(root, string.Empty, string.IsNullOrEmpty(indent) ? string.Empty : indent);
+            dumper.DoDumpXML(root, string.Empty, string.IsNullOrEmpty(indent) ? string.Empty : indent!);
```

Comment:
>string [](start = 49, length = 6)

Consider using `RoslynString.IsNullOrEmpty` to avoid the suppression.

Reply:
alternatively this specific scenario seems like it would be improved by `indent ?? string.Empty`